sys/shell/commands/sc_netif.c

```diff
@@ -218,14 +218,14 @@ static void _netif_set_u16(kernel_pid_t dev, ng_netconf_opt_t opt,
         if ((res = strtoul(u16_str, NULL, 10)) == ULONG_MAX) {
             puts("error: unable to parse value.\n"
                  "Must be a 16-bit unsigned integer (dec or hex)\n");
-            return;
+            return 1;
```

Comment:
I was just wondering, is there a reason you're not returning ``EINVAL`` here (an other errnos in the rest of the document)?